engine/src/main/java/org/hibernate/search/util/impl/ClassLoaderHelper.java

```diff
@@ -69,28 +75,42 @@ private ClassLoaderHelper() {
 	 * @return a new instance of classToLoad
 	 *
 	 * @throws SearchException wrapping other error types with a proper error message for all kind of problems, like
-	 * missing proper constructor, wrong type, security errors.
+	 * missing proper constructor, wrong type, securitymanager errors.
```

Comment:
`security manager` with a space would be better, wouldn't it?

Reply:
I'm referring specifically to the securitymanager system of the JVM, not a generic "security manager". I changed it as the previous one was a bit alarming but it's not very important.